README.md

```diff
@@ -250,32 +250,32 @@ git push origin <branch-name>
 ### Slightly more advanced concepts
 
 There is much more to Git and GitHub, herebelow we will introduce some more
-advanced concepts. If they often prove very useful, they are not absolutely
+advanced concepts. While they often prove very useful, they are not absolutely
 necessary, so maybe don't focus on these too much at the beginning.
 
 #### Issues
 
-Issues describe bugs, wanted features, questions, requests about a repository.
+Issues describe bugs, wanted features, questions, and requests about a repository.
 To create an issue, go to the `Issues` tab on the repository's page on GitHub,
 and click `New issue`.
 
 <p align="center">
   <img alt="Create an issue" src="assets/create-issue.png"/>
 </p>
 
-Then you can assign people to the issue, add a description and a label to
+Then you can assign people to the issue, add a description and add a label to
 categorize your issue (is it a bug report, a request for a feature, a question,
 ...?). The repository contributors can then discuss the issue here.
 
 #### Branches
 
 Very early, you might encounter `conflicts`. These happen when you worked on
-the same files and lines as someone else. It will usually occur while pulling or
+the same files and lines of code as someone else. It will usually occur while pulling or
 pushing, as you try to synchronize your local work with the online repository.
```

Comment:
```suggestion
the same files and lines of code as someone else. It will usually occur while
pulling or pushing, as you try to synchronize your local work with the online
repository.
```